internal/pkg/config/v1/config.go

```diff
@@ -133,6 +134,25 @@ func New() *Config {
 	}
 }
 
+func (c *Config) DecryptContextStates() error {
+	if c.CurrentContext != "" {
+		context := c.Context()
+		state := c.ContextStates[context.Name]
```

Comment:
We can't use `context.State` here?

Reply:
This was copied from the original code:
`context.State = c.ContextStates[context.Name]` so i think it's different